api.json

```diff
@@ -152,7 +140,11 @@
     "properties": [
       {
         "name": "permissionNative",
-        "type": "'granted' | 'denied' | 'default'"
+        "type": "NotificationPermission"
+      },
+      {
+        "name": "permission",
+        "type": "boolean | undefined"
```

Comment:
Is the `undefined` here correct for `permission`?  `NotificationPermission` above don't have `undefined` it's definition so one of these two are not correct.